test/index.js

```diff
@@ -0,0 +1,57 @@
+const proxyquire = require('proxyquire').noCallThru()
```

Comment:
the real AWS package gets called if I don't add `.noCallThru()`, but I don't know why